{"uuid": "15d041f7-1050-48a2-a915-fde11e7cce98", "counter": 5, "stdout": "\u001b[1;31mfatal: [10.100.0.120]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: Load key \\\"/tmp/tmp0f3nlcuw\\\": error in libcrypto\\r\\nuser_diaken@10.100.0.120: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).\", \"unreachable\": true}\u001b[0m", "start_line": 4, "end_line": 5, "runner_ident": "0492a69a-d11b-4297-9eb0-af53e6144932", "event": "runner_on_unreachable", "pid": 231437, "created": "2025-05-17T23:42:30.749015+00:00", "parent_uuid": "005056b7-f67d-b757-e113-000000000010", "event_data": {"playbook": "/tmp/tmp_51eamns.yml", "playbook_uuid": "36ff6cc2-0910-4712-bf33-c2c09a2dab99", "play": "Revisar logs del sistema y autenticaci\u00f3n", "play_uuid": "005056b7-f67d-b757-e113-000000000002", "play_pattern": "target_host", "task": "Gathering Facts", "task_uuid": "005056b7-f67d-b757-e113-000000000010", "task_action": "gather_facts", "resolved_action": "ansible.builtin.gather_facts", "task_args": "", "task_path": "/tmp/tmp_51eamns.yml:2", "host": "10.100.0.120", "remote_addr": "10.100.0.120", "start": "2025-05-17T23:42:30.633243+00:00", "end": "2025-05-17T23:42:30.748609+00:00", "duration": 0.115366, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: Load key \"/tmp/tmp0f3nlcuw\": error in libcrypto\r\nuser_diaken@10.100.0.120: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).", "changed": false}, "uuid": "15d041f7-1050-48a2-a915-fde11e7cce98"}}